src/components/templates/PosterHall/components/PosterHallSearch/PosterHallSearch.tsx

```diff
@@ -30,6 +32,7 @@ export const PosterHallSearch: React.FC<PosterHallSearchProps> = ({
         placeholder="Search Posters/Demos"
         value={searchInputValue}
         onChange={onInputFieldChange}
+        register={ALWAYS_NOOP_FUNCTION}
```

Comment:
[NOTE]

Because of `register={ALWAYS_NOOP_FUNCTION}` there are few forms that need to be tested. Usually forms wouldn't rely on handlers like `onChange()` and there is probably a way to declare either `onChange` or `register` prop be present, but for now, here is the list:

![sparkle-mandatory-register-01](https://user-images.githubusercontent.com/79229621/155150084-85669955-23c4-4a16-8c92-ade4b3f32a04.png)

Reply:
thanks!